test/unit/exports/lib_b/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+// The functionality from `lib_a` should be usable within this crate
```

Comment:
Functionality from`lib_a` will be available within this crate even without `pub use`. Did you want to say lib_b re-exports symbols from lib_a (so users of lib_b can use symbols from lib_a directly)?

Reply:
correct